src/HighlightCeption.php

```diff
@@ -25,13 +25,13 @@ class HighlightCeption extends Module
         'timeWait' => 1,
         'module' => 'WebDriver'
     ];
-
-    /**
-     * Css style for hightlight text or element
-     *
-     * @var string
-     */
-    private $cssStyle = "";
+	
+	/**
+	 * Name to use for css style we inject in page
+	 *
+	 * @var string
+	 */
+	 private $styleName = "hlght";
```

Comment:
please make `$styleName` more generic;
- dynamically created name, eg `$this->styleName = 'hlght'.rand(0,1000);` (or something like that)
- or allow change these variable via configuration

Reply:
I liked your idea of making it configurable. This way the user can always change it themselves should it ever be a problem.